hieroplatform/src/main/java/org/hiero/sketch/spreadsheet/BasicColStat.java

```diff
@@ -0,0 +1,121 @@
+package org.hiero.sketch.spreadsheet;
+
+import org.hiero.sketch.table.api.IColumn;
+import org.hiero.sketch.table.api.IMembershipSet;
+import org.hiero.sketch.table.api.IRowIterator;
+import org.hiero.sketch.table.api.IStringConverter;
+import javax.annotation.Nullable;
+
+public class BasicColStat {
+    private final int MOMENT_NUM;
```

Comment:
In general we use all caps only for enums and constants. I would suggest camelCase: momentCount.